settings/prod.py

```diff
@@ -6,8 +6,9 @@
                  'art-backend.herokuapp.com']
 
 CORS_ORIGIN_WHITELIST = (
-    'art-dashboard-staging.herokuapp.com'
+    'art-dashboard-staging.herokuapp.com',
+    'staging-art.andela.com'
 )
```

Comment:
we can exclude the staging ones here

Reply:
@egichuri  should I remove even the art heroku staging?

Reply:
yeah.. let's remove everything staging from prod.py